pkg/cloud/tags/tags_test.go

```diff
@@ -34,14 +34,7 @@ import (
 )
 
 var (
-	bp = infrav1.BuildParams{
```

Comment:
I inlined the `BuildParams` object on each test case to make it easier to see what's being tested.